index.html

```diff
@@ -6,8 +6,85 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Assignment 1</title>
+    <script src="login.js"></script>
 </head>
+
 <body>
-    <h1>comproDLS</h1>
+    <main>
+        <div class="container">
+            <div class="container__left">
+                <div class="info-area">
+                    <p class="info-area__h1">
+                        comproDLS<span>TM</span>
+                    </p>
+                    <p class="info-area__h2">
+                        OnBoarding
+                    </p>
+                    <hr id="info-area__hr">
+                    <p class="info-area__h3">
+                        Application
+                    </p>
+                    <p class="info-area__text mt-2">
+                        Welcome to the comproDLS<span>TM</span> OnBoarding 
+                        Application for mentors and freshers. Please sign in using your organization email and password.
+                    </p>
+                    <p class="info-area__text mt-1">
+                        In case you do not have an account, click on 'Sing Up now' to get started.
+                    </p>
+                </div>
+            </div>
+            <div class="container__right">
+                <div class="login-area">
+                    <p class="signup web-visi">
+                        Not a member? &nbsp;&nbsp;<span><a href="#">
+                            Request an Account</a>
+                        </span>
+                    </p>
+    
+                    <div class="login-box">
+
+                        <div class="msg-box" id="message_box">
```

Comment:
we an try use single element here instead of `msg-box` & `msg-box__message` and style that accordingly. (would need to update js code as well)